feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.spec.ts

```diff
@@ -55,6 +55,17 @@ class MockProductService {
   }
 }
 
+@Component({
+  // tslint:disable-next-line: component-selector
```

Comment:
is this really required? in other test this is not used, although they are similar

Reply:
Removed